### PR TITLE
tests: more cleanup of plugin/shada_spec

### PR DESCRIPTION
**Pull request by blueyed**
Ref: https://github.com/neovim/neovim/pull/10701/commits/330a6713bf888b1b612a1ebebb5d95bb0b31cd53#r311005754

### CODE REVIEW

File: test/functional/plugin/shada_spec.lua [hunk 1]
```diff
@@ -13,14 +13,8 @@ local shada_helpers = require('test.functional.shada.helpers')
 local get_shada_rw = shada_helpers.get_shada_rw
 
 local function reset(shada_file)
-  -- TODO(justinmk): why is this needed?
-  local rtp_value = ('\'%s/runtime\''):format(
-      paths.test_source_path:gsub('\'', '\'\''))
```
Comment (justinmk) [inline test/functional/plugin/shada_spec.lua]: The TODO was part curiosity about what was intended and whether it's useful for clear() in general.

**Review by justinmk — APPROVED** [review]

File: test/functional/plugin/shada_spec.lua (hunk 1 above)
Comment (blueyed) [reply test/functional/plugin/shada_spec.lua]: Moved it down with the laststatus.  ok I guess?
`reset` here appears to feature havin a specific fname, which is decent I assumed.

File: test/functional/plugin/shada_spec.lua [hunk 2]
```diff
@@ -2554,6 +2548,8 @@ describe('syntax/shada.vim', function()
   it('works', function()
     nvim_command('syntax on')
     nvim_command('setlocal syntax=shada')
+    -- TODO(justinmk): why is this needed?
+    nvim_command('set laststatus&')
```
Comment (blueyed) [inline test/functional/plugin/shada_spec.lua]: Still wondering why this is needed myself, and how you came up with it?
(https://github.com/neovim/neovim/pull/10701#discussion_r311005754)

Comment (justinmk) [reply test/functional/plugin/shada_spec.lua]: I just compared `helpers.nvim_args`.

File: test/functional/plugin/shada_spec.lua (hunk 1 above)
Comment (justinmk) [reply test/functional/plugin/shada_spec.lua]: I meant the `paths.test_source_path` rtp value.

File: test/functional/plugin/shada_spec.lua (hunk 2 above)
Comment (blueyed) [reply test/functional/plugin/shada_spec.lua]: Ah.  Should I remove the TODO here then?
Might be good to have an explanation why it is needed (if you happen to have an idea).

Comment (justinmk) [reply test/functional/plugin/shada_spec.lua]: it's just needed to make the test pass. Don't know why, almost certainly just how shada works. Could adjust the test somehow bit didn't want to spend time on it.

**Pull request closed without merging**